com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/resources/CFExceptionMapper.java

```diff
@@ -35,6 +36,11 @@
         if (e instanceof ConflictException) {
             status = HttpStatus.CONFLICT;
         }
+        if (e instanceof ResponseStatusException) {
```

Comment:
Add a test case for this in the unit test.

Reply:
Done.